packages/experimental/RadioGroup/src/Radio/useRadio.win32.ts

```diff
@@ -51,10 +52,59 @@ export const useRadio = (props: RadioProps): RadioInfo => {
     }
   }, []);
 
+  const isCircularNavigation = true;
+
+  const onInvoke = React.useCallback(
+    (e: KeyPressEvent) => {
+      if (
+        e.nativeEvent.key == 'ArrowDown' ||
+        e.nativeEvent.key == 'ArrowRight' ||
+        e.nativeEvent.key == 'ArrowUp' ||
+        e.nativeEvent.key == 'ArrowLeft'
+      ) {
+        const length = radioGroupContext.enabledValues.length;
+        const currRadioIndex = radioGroupContext.enabledValues.indexOf(radioGroupContext.value);
+        let newCurrRadioIndex;
+        if (e.nativeEvent.key === 'ArrowDown' || e.nativeEvent.key == 'ArrowRight') {
+          if (isCircularNavigation || !(currRadioIndex + 1 == length)) {
+            newCurrRadioIndex = (currRadioIndex + 1) % length;
```

Comment:
Since we know (currRadioIndex + 1) is less than length, do we need the %? I don't think it changes anything?

When X < Y, X % Y = X always (I believe?)

Reply:
Reworked the logic a little bit by removing the check for length, so I'll need to keep the % for circular navigation to work correctly.